django_admin_keyboard_shortcuts/templates/admin/base.html

```diff
@@ -1,7 +1,5 @@
 {% extends "admin/base.html" %}
-{% load static %}
-{% load i18n %}
-{% load shortcuts %}
+{% load static i18n shortcuts %}
```

Comment:
Personal preference - we can continue to load separately if you prefer?)

Reply:
Fine for me, will get sorted out in the merge to Django.

I like them on separate lines for smaller diffs when you change them, but I think Django prefers your style.